src/cgr_gwas_qc/parsers/plink.py

```diff
@@ -73,7 +74,7 @@ def read_hwe(filename: PathLike) -> pd.DataFrame:
     )
 
 
-def read_genome(filename: PathLike) -> pd.DataFrame:
+def read_genome(filename: PathLike, required_cols=None, chunksize=1000000) -> pd.DataFrame:
```

Comment:
The whole docstring here needs updated. Here are somethings in the docstring that need fixed:
* We are no longer applying sorting in this function, so we should not say we are here.
* `A (n x 5) table with ...` is wrong. It is a `(n x 17)`.
* `ID1, object` should be`ID1, string`. Same for `ID2`
*  `RT, object,` should be `RT, category,`. Same for `EZ`
* `PHE, int` should be `PHE, category`
* `HOMHOM, float,` should be int. Same for `HETHET`
*

Reply:
Updated in 9b57d4395aaaf5eabda8ad3b1d218dc76a358490